main/src/main/java/com/airbnb/reair/incremental/db/PersistedJobInfoStore.java

```diff
@@ -411,6 +278,81 @@ public void run() throws Exception {
     }
   }
 
+  private synchronized void batchPersistNewImpl(List<PersistedJobInfo> jobs)
```

Comment:
Nit: Name seems odd

Reply:
semi-standard convention for dummy helpr functions (this one is because of knowing which exceptions are thrown)